youtube/quickstart.js

```diff
@@ -0,0 +1,131 @@
+var fs = require("fs")
+var readline = require("readline")
+var { google } = require("googleapis")
+var OAuth2 = google.auth.OAuth2
+
+// If modifying these scopes, delete your previously saved credentials
+// at ~/.credentials/youtube-nodejs-quickstart.json
+var SCOPES = ["https://www.googleapis.com/auth/youtube.force-ssl"]
+var TOKEN_DIR =
```

Comment:
Can we update the path here to generate credentials inside youtube folder instead of root

Reply:
Understood, just pushed an update here : https://github.com/DevC-Casa/geeksblabla.com/pull/305/commits/ef8ea248a8e9d5da91ff7b3536f0595c327920c7

I should mention that this file is a slightly modified copy of [this one](https://developers.google.com/youtube/v3/quickstart/nodejs), hence the callback hell and outdated practices